components/ui/ViewAllLeadsButton/ViewAllLeadsButton.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import Image from 'next/image';
+import exportImage from '/assets/images/leadslist-icons/export.png';
+import { useRouter } from 'next/navigation';
+import React from 'react';
+
+function ViewAllLeadsButton() {
+  const router = useRouter();
+  const handleCreateLeadButtonClick = () => {
+    router.push('/leads/');
+  };
```

Comment:
You should keep a line-break after this line.

Reply:
Done